lib/controllers/root-controller.js

```diff
@@ -362,13 +367,15 @@ export default class RootController extends React.Component {
   }
 
   @autobind
-  async initializeRepo() {
+  async initializeRepo(initDialogPath) {
     if (this.props.activeWorkingDirectory) {
       await this.acceptInit(this.props.activeWorkingDirectory);
-      return;
+      return null;
```

Comment:
Good points. 

After discussion with @BinaryMuse and @smashwilson we decided it’s probably best to drop this conditional and always prompt the user for the directory they want to initialize a repo in. 

Since we’re pre-filling the input for them it’s more like a quick confirmation. And it makes it very explicit where we’ll be initializing the repo. This can be helpful in situations where it’s not quite obvious to the user that we’ll be initializing a repo for the folder that the active item belongs to (say they have multiple panes with active items belonging to different folders)